keys/cryptostore/encoder_test.go

```diff
@@ -18,22 +18,22 @@ func TestNoopEncoder(t *testing.T) {
 	key := cryptostore.GenEd25519.Generate(cmn.RandBytes(16))
 	key2 := cryptostore.GenSecp256k1.Generate(cmn.RandBytes(16))
 
-	b, err := noop.Encrypt(key, "encode")
+	_, b, err := noop.Encrypt(key, "encode")
 	require.Nil(err)
 	assert.NotEmpty(b)
 
-	b2, err := noop.Encrypt(key2, "encode")
+	_, b2, err := noop.Encrypt(key2, "encode")
 	require.Nil(err)
 	assert.NotEmpty(b2)
 	assert.NotEqual(b, b2)
 
 	// note the decode with a different password works - not secure!
-	pk, err := noop.Decrypt(b, "decode")
+	pk, err := noop.Decrypt([]byte{}, b, "decode")
```

Comment:
You can pass in `nil` here, can't you? Works like []byte{} for anything that just reads.